packages/frontend/src/utils/timestamp-text-util.ts

```diff
@@ -26,17 +26,16 @@ export const mapTimeStampToLabel = (
 
 /**
  * Finds the week of year that the given timestamp is in.
- * Source: https://www.geeksforgeeks.org/calculate-current-week-number-in-javascript/
  * @param {string} timestamp
```

Comment:
Dont need type when typescript. Nor the parameter itself. 
Jsdoc should just be. Maybe even specified the timestamp formatting that is preferrable.

```typescript
/**
 * Finds the week of year that the given timestamp is in.
 * Source: https://www.geeksforgeeks.org/calculate-current-week-number-in-javascript/
 */
```